File: MApps/MCtripModules/MCtripDeviceModule/MCtripDeviceIosModule/src/main/java/com/mctrip/modules/device/ios/IosFragment.java
package com.mctrip.modules.device.ios;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.Toast;

import com.mctrip.library.base.MFragment;


public class IosFragment extends MFragment {
    @SuppressLint("InflateParams")
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View contentView = inflater.inflate(R.layout.fragment_ios, null);
        Button buttonIos = (Button) contentView.findViewById(R.id.buttonIos);
        buttonIos.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(mFragmentActivity, "this is ios child module", Toast.LENGTH_SHORT).show();
            }
        });
        return contentView;
    }
}

File: android/app/modules/mobile.ios/src/main/java/com.multiapk.modules.mobile.ios/IosFragment.kt
package com.multiapk.modules.mobile.ios

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import com.jakewharton.rxbinding2.view.RxView
import com.multiapk.library.base.CMFragment
import org.jetbrains.anko.toast
import java.util.concurrent.TimeUnit


class IosFragment : CMFragment() {
    @SuppressLint("InflateParams")
    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val contentView = inflater!!.inflate(R.layout.fragment_ios, null)
        val buttonIos = contentView.findViewById(R.id.buttonIos) as Button
        RxView.clicks(buttonIos).throttleFirst(1, TimeUnit.SECONDS).subscribe {
            context?.toast("this is ios child module")
        }
        return contentView
    }
}
